pypeit/spectrographs/spectrograph.py

```diff
@@ -86,6 +86,12 @@ def default_sky_spectrum():
     @staticmethod
     def default_pypeit_par():
         return pypeitpar.PypeItPar()
+    
+    def get_lacosmics_par(self,proc_par,binning=None):
```

Comment:
This is a pretty messy work-around.

We should discuss a better approach, right @kbwestfall ?

Reply:
Please suggest one.

Reply:
See my discussion in the #binning channel in slack. I'm leaving this as is for now.